sdk-docs/project-structure.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Project Structure
+category: 61fcd8e1a448f5004215317c
+---
+
+> 🚧 Beta
+> 
+> This is a beta version and subject to change without notice. Pricing, terms, conditions and availability may change in the final version.
+
+# Project Structure
+The Trackunit SDK leverages the Nx build system to generate a mono-repo structure for app development. This structure allows developers to create and manage multiple apps and extensions within the same repository. By utilizing the mono-repo approach, developers can easily share code and resources between apps, reducing redundancy and enhancing maintainability. The shared codebase promotes consistency across applications and simplifies collaboration within teams. Therefore our suggested approach is to have *one* workspace setup and develop all your apps and extensions in it.
+## Table of Contents
+1. [Overview](#overview)
+2. [Main Directories](#main-directories)
+3. [Configuration Files In Depth](#configuration-files)
+
+## Overview
+### Difference between apps and extensions
+It is important to distinguish between an app and an extension as it might be confusing in the beginning.
+An "app" serves as a container in the marketplace, bundling one or more "extensions" to create an installable unit. The app handles configuration and metadata such as appearance in the marketplace, installation process, and descriptive information.
+
+On the other hand, an "extension" is where the main programming efforts are directed, as it contains the essential application logic and user interface. It is, in fact, the core functionality of the app.
```

Comment:
While this is not at all wrong, it just felt a bit odd that you came from talking about what the app IS and then moves on to tell that an extension is where one should direct its efforts..